data/NBO/Trp_polar.inp

```diff
@@ -0,0 +1,35 @@
+! hf sto-3g nori verytightscf noautostart NBO
```

Comment:
Can we make this directory more specific?  Something like `data/NBO/basicNBO7.0_basicORCA5.0/...`?  (There's no precedence for this nesting AFAIK.)

Reply:
I have introduced your suggested structure as it is better than just NBO top level, but certainly something we can discuss. The only other option would be to group them by two directories `basicNBOversion/basicCallingCodeversion/`

Reply:
My email comment didn't work for some reason, I mean that this idea (`X/Y/`) is better than mine (`X_Y/`).